### PR TITLE
branch for healthbar

### PR DESCRIPTION
**Pull request by dylanhollohan**

### CODE REVIEW

**Comment by jess-zhai** [comment]
@dylanhollohan it should be good to merge to main without conflict. Thanks!

File: Assets/Scripts/PlayerMovement.cs [hunk 1]
```diff
@@ -0,0 +1,83 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerMovement : MonoBehaviour
```
Comment (dylanhollohan) [inline Assets/Scripts/PlayerMovement.cs:5]: Is this class just for player movement in your own scene?  If so, can you add some comments about that like a javadoc or maybe even rename it?  either one is fine.

File: Assets/Materials/InputMaterial.mat [hunk 2]
```diff
@@ -116,8 +116,8 @@ Material:
     - _WorkflowMode: 0
     - _ZWrite: 1
     m_Colors:
-    - _BaseColor: {r: 1, g: 0, b: 0, a: 1}
-    - _Color: {r: 1, g: 0, b: 0, a: 1}
+    - _BaseColor: {r: 0, g: 1, b: 0, a: 1}
```
Comment (dylanhollohan) [inline Assets/Materials/InputMaterial.mat:119]: I usually roll back the changes to input material before I push, but it's fine for now.  It seems to just save whether the input was red or green before ending the game or something like that.

**Review by dylanhollohan — COMMENTED** [review]
Looks good to me, I just had some comments about documentation for the PlayerMovement, if it's only being used in your scene

File: Assets/Scripts/PlayerMovement.cs (hunk 1 above)
Comment (jess-zhai) [reply Assets/Scripts/PlayerMovement.cs:5]: I merged main into health_bar. Everything should work fine with the version in health_bar (where this file is no longer used). So this file could be deleted, but I'll add a comment if we're still keeping it.

Comment (dylanhollohan) [reply Assets/Scripts/PlayerMovement.cs:5]: oh, okay let's leave it til we double check that nobody else is using it.  Sounds like you and I aren't using it though.

**Review by jrdn-l — APPROVED** [review]

**Pull request merged**